session_items.py

```diff
@@ -1,73 +1,111 @@
 from flask import session
+import os
+import requests
+
+from Task import Task, Status
 
 _DEFAULT_ITEMS = [
     { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
     { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
 ]
 
+BOARD_ID = os.environ.get('BOARD_ID')
+TRELLO_KEY = os.environ.get("TRELLO_KEY")
+TRELLO_TOKEN = os.environ.get("TRELLO_TOKEN")
+
+TODO_LIST_ID = os.environ.get("TODO_LIST_ID")
+DONE_LIST_ID = os.environ.get("DONE_LIST_ID")
+
+trello_baseurl = "https://api.trello.com/"
+trello_apiversion = '1'
+trello_commonurl = trello_baseurl + trello_apiversion
+boardselector = "/board/"
+listselector = "/lists/"
```

Comment:
Given these 2 are only ever used once I'd be tempted just to use string interpolation to keep things readable `f'...'`

Reply:
I think I prefer the consistency of doing it the same for `/cards/` `/lists/` and `/board/` , and `/cards/` comes up a few times